index.ts

```diff
@@ -1,4 +1,15 @@
-const EOL = '\r\n';
+
+let newLine = '\r\n';
+const isNode = new Function('try {return this===global;}catch(e){return false;}');
```

Comment:
Was ist denn der Grund, hier den Function-Body als string zu übergeben und nicht einfach folgendes?

```ts
function isNode () {
  try {
    return this === global
  }
  catch {
    return false;
  }
}
```

Als alternative kann man afaik auch einfach prüfen, ob `process` undefined ist (Passiert in z. 7 ja auch schon)